tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/view_image.py

```diff
@@ -16,9 +16,8 @@ class ViewImage(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == 'Gallery')
+        self.wait_to_be_displayed()
```

Comment:
Shouldn't this be Gallery(self.marionette).wait_to_be_displayed()? (and do the gallery import before that then, of course).